### PR TITLE
Script_for_Pre-system_Leaves

### PR DESCRIPTION
**Pull request by AbdoHamed-TheBadNomads**

### CODE REVIEW

File: Pre_System_Inerstion.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Utilities
+import pandas as pd
+import os
+
+from datetime import datetime, timedelta
+from Member import member_db 
+from Leave import leave_db
+
+data = pd.read_csv(os.getenv("Pre_System_Leaves_excel_Path"))
```
Comment (AntWong-TheBadNomads) [inline Pre_System_Inerstion.py:9]: can we just make this such that we input a path?

Comment (AbdoHamed-TheBadNomads) [reply Pre_System_Inerstion.py]: Done

**Review by MabroukTBN — COMMENTED** [review]
Misspelling in file name, _Insertion_ not _Inerstion_

**Comment by AbdoHamed-TheBadNomads** [comment]
> Misspelling in file name, _Insertion_ not _Inerstion_

Done

**Pull request merged**